Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tozd/runit
+FROM tozd/runit:ubuntu-bionic
```

Comment:
The `latest` is in fact equivalent to `ubuntu-trusty`. This is a backwards compatible issue. We should not be using that tag at all, but there it is. As you see, we do not yet use the explicit label everywhere.

So if you have not tested the image after changing this, change it to `ubuntu-trusty`.

Also, feel free to rename this file then to `ubuntu-trusty.dockerfile`.